### PR TITLE
Moved format specific testing into formats package

### PR DESCRIPTION
**Pull request by lolletsoc**
@kbairak

### CODE REVIEW

**Comment by kbairak** [comment]
@lolletsoc is this ready?

**Comment by lolletsoc** [comment]
No, I'm writing some tests for the handler. I'll remove the DNM when it is.

**Comment by lolletsoc** [comment]
@kbairak I've added as much as I believe I should at this point. I could literally spend the rest of the sprint writing tests.

File: openformats/tests/formats/plaintext/test_plaintext.py [hunk 1]
```diff
@@ -1,9 +1,10 @@
 import unittest
-from openformats.tests.common import CommonFormatTestCase
+
+from openformats.tests.formats.common import CommonFormatTestCase
 from openformats.formats.plaintext import PlaintextHandler
 
 
 class PlaintextTestCase(CommonFormatTestCase, unittest.TestCase):
     FORMAT_EXTENSION = "txt"
```
Comment (kbairak) [inline openformats/tests/formats/plaintext/test_plaintext.py]: I think I made the extension part of the handler itself

**Pull request merged**